src/mlpack/methods/ann/layer/select_impl.hpp

```diff
@@ -28,6 +28,50 @@ Select<InputDataType, OutputDataType>::Select(
     // Nothing to do here.
   }
 
+template <typename InputDataType, typename OutputDataType>
+Select<InputDataType, OutputDataType>::Select(
+    const Select& layer) :
+    index(layer.index),
+    elements(layer.elements)
+{
+   //Nothing to do here     
```

Comment:
```suggestion
   // Nothing to do here     
```

Pedantic style issue.